SAP/spartacus
# cpq Discount Percentage

Discount Percentage Library
![image](https://github.com/SAP/spartacus/assets/164467950/ed926f6c-3409-4d42-bb23-02e714f39fb4)

https://jira.tools.sap/browse/CXSPA-7040
**How to test**

- Change your base URL to the S1 API URL: https://api.cg79x9wuu9-eccommerc1-s1-public.model-t.myhybris.cloud/
- Start the server
- Log in with user ID as james.weber@harvestlive.inc and password as 'welcome'
- Navigate to quote details for Quote ID 02791917
- Inside the Quote Details page, check the Card table; the Discount Percentage Column will be available.



The following is a checklist for new epics or features acceptance for Spartacus, based on [our Definition of Done document](https://sap.github.io/spartacus-docs/definition-of-done/)

## General

- [ ] Epic has been tested on a production-like server using Spartacus installation script
- [x] Epic has been tested in all supported browsers and devices (Check list on DoD page)
- [x] Epic has sufficient end to end (cypress) test coverage
- [ ] Schematics code deprecations have been provided and tested
- [x] Epic complies with [Security best practices](https://sap.github.io/spartacus-docs/security-best-practices/)
- [x] Epic complies with [Accessibility best practices](https://sap.github.io/spartacus-docs/a11y-best-practices/)
- [x] Epic CSS supports directionality (if required)
- [x] Epic code scans have been conducted [Compliance scans for branches](https://wiki.one.int.sap/wiki/display/spar/Compliance+scans+for+branches)
- [x] Epic renders successfully in SSR

## Audits/reviews

- [ ] Architecture review of the epic
- [ ] Security and threat modeling audit/review from the Security team
- [ ] Accessibility audit/review from the UX team
